components/ExportMenu.js

```diff
@@ -17,11 +18,18 @@ const toIFrame = url =>
 </iframe>
 `
 
-class ExportMenu extends React.PureComponent {
+class ExportMenu extends React.Component {
   state = {
     isVisible: false
   }
 
+  shouldComponentUpdate(nextProps, nextState) {
```

Comment:
Sorry to get back to you late on this @raboid can you avoid using `react-addons-shallow-compare`? I want to actually eliminate that dependency from this code base.

An easy alternative would be to extract the pieces that are `this.props.router.asPath` and then use [`React.memo`](https://reactjs.org/docs/react-api.html#reactmemo) to only update that piece when `this.props.router.asPath` updates